src/components/note/Note.svelte

```diff
@@ -77,13 +77,19 @@
 					<time datetime={note.data.timestamp.toISOString()} class="font-mono text-2.6 c-remark">{Time(note.data.timestamp)}</time>
 				</div>
 				<span class="flex items-center gap-1 sm:ml-a c-remark">
-					{#each note.data.tags as tag}
+					{#each note.data.tags || [] as tag}
```

Comment:
It seems fine in tests without a fallback, but if you add one, prefer `??` instead of `||` since tags can only be `undefined`, not other falsy values.

```suggestion
					{#each note.data.tags ?? [] as tag}
```